example/user-guides/alerting/alertmanager-config-example.yaml

```diff
@@ -5,13 +5,52 @@ metadata:
   labels:
     alertmanagerConfig: example
 spec:
+  inhibitRules:
+  # critical > warning > info
+  #
+  # If an alert with severity 'warning'
+  # and an alert with severity 'info' occur at the same time,
+  # the alerts with severity 'info' will be ignored.
```

Comment:
```suggestion
  # If an alert with severity 'warning' occurs at the same time as an alert 
  # with severity 'critical' and both alerts share the same job label, the former will be muted.
```

Reply:
I modified it.
It looks much better than the first time. Thank you